pages/index.tsx

```diff
@@ -1,17 +1,28 @@
 import Head from "next/head";
-import styles from "../styles/Home.module.css";
-import Button from "../components/loginButton/loginButton";
+import Link from "next/link";
+import Logo from "../components/Logo/logo";
+import styles from "../styles/WelcomePage.module.css";
+
+export default function WelcomePage() {
+  function myFunction() {
```

Comment:
You are defining this function but not calling it in your code.
You could try calling this function inside of a 'useEffect', so it executes once the index page is visited by the user. 

I would also use the 'useRouter' hook instead of location.href.
And rename the function so it describes what it does :)

Reply:
I've used it like you suggested it. and it works 🎉